pkg/tasktesting/run.go

```diff
@@ -60,7 +60,7 @@ type TaskRunContext struct {
 	CollectedLogs []byte
 }
 
-func RunTask(t *testing.T, testOpts TestOpts, taskWorkspaces map[string]string, testCaseContext *TaskRunContext, tc TaskRunCase) {
+func runTask(t *testing.T, testOpts TestOpts, taskWorkspaces map[string]string, testCaseContext *TaskRunContext, tc TaskRunCase) {
```

Comment:
I think the idea originally was that you could use this from other repos but might not work anyway yet. See #163